best-practices/native/react-native/deployment/ios/README.md

```diff
@@ -1,79 +1,26 @@
-	- No alpha channel allowed
-
-4. Select an application category
-
-5. Add short and long descriptions of your application
-
-6. Provide a privacy policy URL
-
-7. Provide a content rating
-	- Google Play offers a series of questions that will automatically provide a content rating for you
-
-### Creating an APK
-
-#### What is an APK?
-The APK is the file that represents your application. It is uploaded to the Google Play Store and is eventually downloaded on the user's phone, letting them run your application. Emulators and dev-mode phones can run unsigned APKs, but you'll need to create a signed APK to distribute your app to users.
-
-#### What is a signed APK?
-A signed APK is an APK that has an attached keystore file. The keystore file contains some additional information about the app and the app uploader. If you're putting the app on the Play Store for the first time, you'll need to create a keystore file. If you're updating an existing app, you'll need the existing keystore before you can update the app on the Google Play Store. **IMPORTANT NOTE:**  Make sure you backup your Keystore after creating it. Once an app is published to the Play Store, it will be impossible to update it if you lose the keystore file or password.
-
-#### Steps to Create a signed APK
-###### In Android Studio:
-- Build -> Generate Signed APK
-- Android Studio will ask for the path to your keystore file and its password. If you don't have one, create one here.
-- Set 'Build Type' to 'release' (make note of the APK Destination Folder)
-- Make sure 'V1' and 'V2' Signature Versions are both checked.
-- Click 'finish'
-- After it's done, the APK will be available in the APK Destination Folder
+1. Open the xcode workspace project.
+2. In upper left-hand of screen, click on `Generic iOS Device`. Select a device you'd like to test on.
+3. Build the app via the play button. Be sure the iOS device is set to a physical apple device
+<img width="399" alt="Screen Shot 2021-07-30 at 2 23 02 PM" src="https://user-images.githubusercontent.com/25145605/127720045-664ec7b6-95e1-43b5-af55-9938b5c10c5d.png">
+3. Once it builds successfully, click on Product on the top nav bar and click on Archive
+<img width="675" alt="Screen Shot 2021-07-30 at 3 13 06 PM" src="https://user-images.githubusercontent.com/25145605/127720075-0c66917e-9cc6-4605-870a-6cf775cc409f.png">
+4. This window should pop up 
+<img width="1255" alt="Screen Shot 2021-07-30 at 2 33 39 PM" src="https://user-images.githubusercontent.com/25145605/127720114-ef963c17-f053-45d6-b7b1-b13eccc16c1c.png">
+5. Click on Validate App
+<img width="1255" alt="Screen Shot 2021-07-30 at 2 33 39 PM copy" src="https://user-images.githubusercontent.com/25145605/127720165-8aa5727c-8f8a-4902-a231-b3a432c58ed8.png">
+6. Click on App Store Connect
+<img width="744" alt="Screen Shot 2021-07-30 at 3 01 09 PM" src="https://user-images.githubusercontent.com/25145605/127720262-5bdf986c-ce2d-481d-8f64-48188b3fc0d0.png">
+7. Then Upload
```

Comment:
This feature is not available until after `validate`. After validation successfully completes, we also need to click on "distribute" and basically go through the same steps again, but this time once its all validate, the option to upload will appear at the end.

Reply:
I would suggest moving #7 to the end of the list.

Reply:
That's right, Validate App has other process and options.
We need to finish validate process then clicking on `Distribute APP` to get `App Store Connect`

Reply:
Done via: 29ddf5bf4a1732f3ad21a7dbde06773755ba16e5